src/styles/price.scss

```diff
@@ -0,0 +1,20 @@
+.price {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 0 16px 16px;
+  align-items: center;
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    color: #616070;
+  }
+
+  &__value {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+    color: $main-text-color;
```

Comment:
It's good practice to use variables for colors to improve maintainability of the code. Since `$main-text-color` is used for `.price__value`, consider using a similar variable for `.price__title` color.